src/cluster_copier.cc

```diff
@@ -16,28 +16,14 @@ namespace grpc_labview {
     // cluster: Pointer to the cluster created by LabVIEW
     void ClusterDataCopier::CopyToCluster(const LVMessage& message, int8_t* cluster)
     {
-        std::map<std::string, int> oneof_containerToSelectedIndexMap;
```

Comment:
Note: This container for tracking selected index of oneof containers has moved to the message class.